crates/ruma-macros/src/events/event_enum.rs

```diff
@@ -450,16 +450,16 @@ fn expand_accessor_methods(
                 match self {
                     #(
                         #self_variants(event) => {
-                            event.as_original().map(|ev| #content_variants(ev.content.clone()))
+                            event.original_content().cloned().map(#content_variants)
```

Comment:
There is a slight issue here with `RoomRedactionEventContent`, contrary to other types, this method return an owned type (because we need to construct an enum), so `.cloned()` doesn't work (and is unnecessary).

Do we make all of these methods return an owned type, or do we just `.clone()` the content of `RoomRedactionEventContent` even if it's unnecessary?

Reply:
I'd say let's add a (private?) method `into_original_content` that consumes `self`.